### PR TITLE
list change to set

### PR DESCRIPTION
**Pull request by nevermore3**

### CODE REVIEW

**Review by yixinglu — APPROVED** [review]

**Review by dutor — APPROVED** [review]
Well done.

**Review by laura-ding — APPROVED** [review]

**Review by CPWstatic — DISMISSED** [review]

File: tests/common/nebula_test_suite.py [hunk 1]
```diff
@@ -486,6 +487,110 @@ def check_empty_result(self, resp):
             empty = True
         assert empty, msg
 
+    @classmethod
+    def compare_vertex(self, vertex1, vertex2):
+        if not isinstance(vertex1, CommonTtypes.Vertex) or not isinstance(vertex2, CommonTtypes.Vertex):
+            assert False, "Error Type {}, {}".format(type(vertex1), type(vertex2))
```
Comment (yixinglu) [inline tests/common/nebula_test_suite.py:493]: assert isinstance() and isinstance()

File: tests/common/nebula_test_suite.py [hunk 2]
```diff
@@ -486,6 +487,110 @@ def check_empty_result(self, resp):
...
+        if vertex1.vid != vertex2.vid:
+            if vertex1.vid < vertex2.vid:
+                return -1
+            return 1
```
Comment (yixinglu) [inline tests/common/nebula_test_suite.py:497]:
```python
if v1 < v2:
  return -1
if v1 > v2:
  return 1
```

File: tests/common/nebula_test_suite.py [hunk 3]
```diff
@@ -486,6 +487,110 @@ def check_empty_result(self, resp):
...
+        if not isinstance(edge1, CommonTtypes.Edge) or not isinstance(edge2, CommonTtypes.Edge):
+            assert False, "Error Type {}, {}".format(type(edge1), type(edge2))
+        if edge1.src != edge2.src:
+            if edge1.src < edge2.src:
+                return -1
+            return 1
+        if edge1.dst != edge2.dst:
+            if edge1.dst < edge2.dst:
+                return -1
+            return 1
```
Comment (yixinglu) [inline tests/common/nebula_test_suite.py:513]: ditto

File: tests/common/nebula_test_suite.py [hunk 4]
```diff
@@ -486,6 +487,110 @@ def check_empty_result(self, resp):
             empty = True
         assert empty, msg
 
+    @classmethod
+    def compare_vertex(self, vertex1, vertex2):
```
Comment (yixinglu) [inline tests/common/nebula_test_suite.py:491]: replace `self` with `cls`

**Review by CPWstatic — APPROVED** [review]

**Review by laura-ding — APPROVED** [review]

**Pull request merged**